internal/kubernetes/operator/project/project.go

```diff
@@ -820,37 +821,23 @@ func toMatcher(m map[string]interface{}) (akov2.Matcher, error) {
 	if len(m) == 0 {
 		return matcher, errors.New("empty map cannot be converted to Matcher")
 	}
-	fieldName, err := keyAsString(m, "fieldName")
+	jsonBytes, err := json.Marshal(m)
 	if err != nil {
-		return matcher, err
+		return matcher, fmt.Errorf("could not marshal mather map %v back to json: %w", m, err)
```

Comment:
funny but probably needs better messaging, also I wonder if helpful to be `%#v` instead of plain `%v`